### PR TITLE
Inserting timestamps in track data.

### PR DESCRIPTION
**Pull request by taiar**
This PR inserts timestamps in tracking data.

The timestamp is created in current time and increased one by one second when pushed.

### CODE REVIEW

**Comment by martinstiago** [comment]
👍

File: app/assets/javascripts/action_tracker.js [hunk 1]
```diff
@@ -2,27 +2,37 @@
...
+    if(typeof cfg_options !== 'undefined') options = cfg_options;
     storage.queue(list);
   }
 
   function process() {
     while(typeof storage.getFirst() !== 'undefined') {
-      var tracker = new Tracker(storage.dequeue());
-      tracker.send();
+      var tracker = new Tracker(storage.dequeue(), trackerOptions());
+      tracker.addTimestamp(time_seed).send();
```
Comment (antoniosb) [inline app/assets/javascripts/action_tracker.js]: Wouldn't be better if we check the timestamp options here? This way, `addTimestamp` will only know about adding timestamp. Also it keeps easier to extract is decision logic out of here, in a further moment. What do you think?

File: app/assets/javascripts/action_tracker.js [hunk 2]
```diff
@@ -2,27 +2,37 @@
...
     }
   }
 
+  function trackerOptions() {
+    return {
+      timestamp: (typeof options.timestamp !== 'undefined') ? options.timestamp : true
```
Comment (antoniosb) [inline app/assets/javascripts/action_tracker.js]: Why `true` is the default option? If this behavior is not related to core, I think it should be false, then if we want to use it, set it. Isn't it right? Will it change the interface?

File: app/assets/javascripts/action_tracker.js [hunk 3]
```diff
@@ -85,6 +96,15 @@ var ActionTracker = function () {
       }
     }
 
+    if(typeof cfg_options !== 'undefined') this.options = cfg_options;
+
+    this.addTimestamp = function(seed) {
+      if(this.options.timestamp)
+        this.data['created_at'] = seed.getTimeSeed();
+      console.log(this.data);
```
Comment (antoniosb) [inline app/assets/javascripts/action_tracker.js]: Why is this necessary? =P

**Comment by tlluiz** [comment]
👍

**Comment by antoniosb** [comment]
:+1:
